regolith/builders/recentcollabsbuilder.py

```diff
@@ -123,9 +123,10 @@ def latex(self):
                 #                print(set([person["name"] for person in people if person]))
                 print(set([person for person in ppl_names]))
             emp = p.get("employment", [{"organization": "missing",
-                                        "begin_year": 2019,
-                                        "co-editors": "what goes here?"}])
+                                        "begin_year": 2019,}])
             emp.sort(key=ene_date_key, reverse=True)
+            if p.get("employment", "position") == "editor":
+                coeditors = p.get("employment", "co-workers")
```

Comment:
I don't think we want a dash in coworkers

Reply:
also, I don't think this is doing what you intend....it will get employment and if it can't find "employment" it will return the string "co-workers"